pkgs/development/python-modules/onnx/default.nix

```diff
@@ -61,6 +66,9 @@ in buildPythonPackage rec {
       --replace 'include(googletest)' ""
     substituteInPlace cmake/unittest.cmake \
       --replace 'googletest)' ')'
+
+    substituteInPlace CMakeLists.txt \
```

Comment:
Could you add a comment to say this is to match the C++ version used to compile abseil, and to say that Onnx 1.15 will have an option to set this with `-DCMAKE_CXX_STANDARD`